### ISSUE TITLE
Give me clear path from Guides forward

### ISSUE THREAD

**Issue by kyyberi** [PlatformOfTrust/collected-feedback#4]
# Describe the Pain you found while using Developer Portal

In the https://developer.oftrust.net/guides/ page I want to know from which Guide you would advice me to start! Now I'm facing shit load of guides and I have no idea from where to start! 

Consider me as integrator who wants to integrate new data source system to Platform of Trust. **Give me the first step to start from and recommend additional guides to follow.** First guide should give me a picture which shows the steps needed to integrate new data source and build a data product out of it. 

Same for application builder persona. 

Might be a good idea to put the paths for integrators and application builders in the front page as button which take you directly to primary guide. At the bottom of the primary guide is links to other recommended guides. 

# Indicate the importance with a number

(1. Critical, 2. Important, 3. Nice-to-have)

**Comment by kyyberi** [comment]
Perhaps this element in front page could be just two guides. One for the integrator (primary guide to start with) and application builder. 

Both need clear paths from the frontage forward. 

The see all guides button can stay there if developer wants to see all guides instead of the initial paths. 

<img width="1280" alt="Näyttökuva 2020-2-23 kello 13 12 57" src="https://user-images.githubusercontent.com/1927158/75111064-6a52e500-563e-11ea-8f79-3c112ace9f78.png">

**Comment by kyyberi** [comment]
Guides should be classified to two: 
1. Master guides
2. Additional supportive guides. 

Supportive guides can be used as additional information (steps to accomplish something) in master guides. 

We have two master guides: 
1) Data source integration (data product 101)
2) Data products in applications (?)

**Comment by Nazarah** [comment]
@kyyberi  We have added new fixes to Developer Portal. Knowledge on the Flow will now Show the following guides:
1. Data Product 101
2. Register an Application
3. Get Bearer Token

This guides will help developers to choose an onboarding flow to the Platform as either Data Provider or Application Developer.

Your request to classify Master and Supportive guides has been added to wishlist. You will be notified once this feature is implemented and released.
